### DISCUSSION TITLE
Importing (small) video files using Turbopack

### DISCUSSION THREAD

**Discussion by danmartens** [vercel/next.js#73181] [Help]
### Summary

I have a project with a few *very* small video files (specifically MP4s) that I'm importing as asset URLs via a custom webpack config:

```javascript
config.module.rules.push({
  test: /\.mp4$/i,
  type: 'asset',
});
```

Is there an equivalent that will work with Turbopack?

Thanks!

**Comment by mwilkins91** [comment]
I would also like to know how to do this...
